modules/st2-login/style.css

```diff
@@ -104,7 +105,7 @@
 
   &-label {
     font-weight: 100;
-
+	font-family: Roboto, sans-serif;
```

Comment:
Nitpick: would be good to use original separator as spaces, not the tabs for consistency with existing codebase

Reply:
Fixed.